src/test/java/core/basesyntax/service/impl/ReportCreatorImplTest.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
```

Comment:
where is the test of valid report creating with valid data? also add tests for null and invalid input data.